### PR TITLE
Call for Content: 2019/14

### PR DESCRIPTION
**Pull request by domenkozar**
Hi all,

please add Nix news item in form of:

```markdown
- [Example news item title](http://example.com)

  Optional item description.

```

preferably as a PR against `master` branch (or a comment below).

Thank you <3

### CODE REVIEW

File: content/2019/14.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Title: #14 - (TODO: title)
+Date: 2019-(TODO: MM-DD)
+Description: (TODO: QUOTE)
+
+# News
+
+- [https://matthewbauer.us/blog/avoid-subshells.html](Improved performance in Nixpkgs)
```
Comment (domenkozar) [inline content/2019/14.md]:
```suggestion
- [Improved performance in Nixpkgs](https://matthewbauer.us/blog/avoid-subshells.html)
```

Comment (matthewbauer) [reply content/2019/14.md]: Thanks!

**Comment by domenkozar** [comment]
- https://jappieklooster.nl/pinning-nixops-builds.html
- https://github.com/saschagrunert/kubernix

**Comment by domenkozar** [comment]
- [Launching Hercules CI](https://blog.hercules-ci.com/2019/10/22/launching-hercules-ci/)

**Comment by globin** [comment]
Link to RFCSC nominations: https://discourse.nixos.org/t/rfc-steering-committee-rotation-2019-20/4589/2

**Comment by domenkozar** [comment]
- [Hercules CI Agent 0.6.1](https://blog.hercules-ci.com/2019/11/12/hercules-ci-agent-0.6.1-release/)

  Performance improvements, a bugfix to IFD and better onboarding experience.

**Comment by domenkozar** [comment]
Thanks all :heart:

**Pull request merged**